### PR TITLE
R4R: Initialize nyancat testnet

### PR DESCRIPTION
**Pull request by zhangyelong**

### CODE REVIEW

File: nyancat/config/genesis.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+          "address": "faa108w3x8",
+          "added_by": "faa108w3x8"
+        }
+      ],
+      "trustees": [
+        {
+          "description": "genesis",
+          "type": "Genesis",
+          "address": "faa108w3x8",
+          "added_by": "faa108w3x8"
```
Comment (chengwenxi) [inline nyancat/config/genesis.json]: Add some profilers and trustees?

Comment (haifengxi) [reply nyancat/config/genesis.json]: Without profilers, we cannot initiate SoftwareUpgrade proposals.

Comment (zhangyelong) [reply nyancat/config/genesis.json]: Done

File: nyancat/config/genesis.json [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+      }
+    ],
+    "auth": {
+      "collected_fee": null,
+      "data": {
+        "native_fee_denom": "iris-atto"
+      },
+      "params": {
+        "gas_price_threshold": "20000000000000",
+        "tx_size": "1000"
```
Comment (haifengxi) [inline nyancat/config/genesis.json]: Let's keep `gas_price_threshold` the same as mainnet.

File: nyancat/config/genesis.json [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        "critical_min_deposit": [
+          {
+            "denom": "iris",
+            "amount": "100"
+          }
+        ],
+        "critical_voting_period": "86400000000000",
+        "critical_max_num": "1",
+        "critical_threshold": "0.8340000000",
+        "critical_veto": "0.3340000000",
```
Comment (haifengxi) [inline nyancat/config/genesis.json]: `critical_threshold` --> 0.857

File: nyancat/config/genesis.json [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+        "critical_participation": "0.8572000000",
+        "critical_penalty": "0.0009000000",
```
Comment (haifengxi) [inline nyancat/config/genesis.json]: `critical_participation` --> 0.875

File: nyancat/config/genesis.json [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+            "amount": "100"
+          }
+        ],
+        "important_voting_period": "86400000000000",
+        "important_max_num": "2",
+        "important_threshold": "0.8000000000",
+        "important_veto": "0.3340000000",
+        "important_participation": "0.8340000000",
+        "important_penalty": "0.0007000000",
+        "normal_deposit_period": "86400000000000",
```
Comment (haifengxi) [inline nyancat/config/genesis.json]: set all three penalty ratios to 0.0

File: nyancat/config/genesis.json [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+        "max_evidence_age": "86400000000000",
+        "signed_blocks_window": "2000",
+        "min_signed_per_window": "0.5000000000",
+        "double_sign_jail_duration": "600000000000",
+        "downtime_jail_duration": "600000000000",
+        "censorship_jail_duration": "600000000000",
+        "slash_fraction_double_sign": "0.0100000000",
+        "slash_fraction_downtime": "0.0000000000",
+        "slash_fraction_censorship": "0.0000000000"
+      },
```
Comment (haifengxi) [inline nyancat/config/genesis.json]: Let's make slashing params the same as mainnet

**Pull request merged**